types/mongration/index.d.ts

```diff
@@ -0,0 +1,35 @@
+// Type definitions for mongration 1.0
+// Project: https://github.com/awapps/mongration#readme
+// Definitions by: Anton Lobashev <https://github.com/soulthreads>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+import { Db } from 'mongodb';
+
+export interface DbConfig {
+    hosts?: string;
+    db?: string;
+    user?: string;
+    password?: string;
+    mongoUri?: string;
+    migrationCollection: string;
+    replicaSet?: string;
+}
+
+export interface MigrationResponse {
+    id: string;
+    status: string;
```

Comment:
Could the type be `"pl" | "skipped" | "error" | "rollback" | "rollback-error" | "not-run"`

Reply:
That's a good suggestion, thanks